portfolio/src/main/webapp/script.js

```diff
@@ -38,6 +32,52 @@ function addRandomOpinion() {
     opinionContainer.innerText = opinion;
 }
 
+function getdata() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+     const listmap = document.createElement("ul");   
+     comments.map(comment => {
+        const list = document.createElement("li");   
+        list.innerHTML = comment;
```

Comment:
One thing to be careful of with using innerHTML is that users might be able to enter bad content. For example what if the user enters this as a comment:

```
<h1>hello!</h1>
```

Or even worse, something like this:

```
<script>alert('hello!')</script>
```

In general I would use innerText instead, unless you do some processing of the content on the server to make sure it's safe.

This can be a pretty complicated topic, so let me know if you want to talk more!